File: Lista1/L1Z2/app/src/main/java/com/example/l1z2/MainActivity.java
package com.example.l1z2;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.view.View;
import android.widget.TextView;
import android.widget.Toast;

import java.util.Random;

public class MainActivity extends AppCompatActivity {

    private boolean bold = false;
    private int wygrane = 0;
    private int przegrane = 0;
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
    }


    private void roll(int choice)
    {
        Random rand = new Random();
        int i = rand.nextInt(3);
        if(i == 0)
        {
            TextView odp = (TextView)findViewById(R.id.odpowiedz);
            odp.setText("Kamień!");

        }
        else if(i == 1)
        {
            TextView odp = (TextView)findViewById(R.id.odpowiedz);
            odp.setText("Papier!");
        }
        else if(i == 2)
        {
            TextView odp = (TextView)findViewById(R.id.odpowiedz);
            odp.setText("Nożyce!");
        }

        if(i == choice)
        {
            Toast.makeText(this,"Remis", Toast.LENGTH_SHORT).show();
        }
        else if(i == 0 && choice == 1)
        {
            Toast.makeText(this,"Wygrałeś!", Toast.LENGTH_SHORT).show();
            wygrane += 1;
            TextView w = (TextView)findViewById(R.id.wygrane);
            w.setText("Wygrane\n" + Integer.toString(wygrane));
        }
        else if(i == 0 && choice == 2)
        {
            Toast.makeText(this,"Przegrałeś :(", Toast.LENGTH_SHORT).show();
            przegrane += 1;
            TextView w = (TextView)findViewById(R.id.przegrane);
            w.setText("Przegrane\n" + Integer.toString(przegrane));
        }
        else if(i == 1 && choice == 0)
        {
            Toast.makeText(this,"Przegrałeś :(", Toast.LENGTH_SHORT).show();
            przegrane += 1;
            TextView w = (TextView)findViewById(R.id.przegrane);
            w.setText("Przegrane\n" + Integer.toString(przegrane));
        }
        else if(i == 1 && choice == 2)
        {
            Toast.makeText(this,"Wygrałeś!", Toast.LENGTH_SHORT).show();
            wygrane += 1;
            TextView w = (TextView)findViewById(R.id.wygrane);
            w.setText("Wygrane\n" + Integer.toString(wygrane));
        }
        else if(i == 2 && choice == 0)
        {
            Toast.makeText(this,"Wygrałeś!", Toast.LENGTH_SHORT).show();
            wygrane += 1;
            TextView w = (TextView)findViewById(R.id.wygrane);
            w.setText("Wygrane\n" + Integer.toString(wygrane));
        }
        else if(i == 2 && choice == 1)
        {
            Toast.makeText(this,"Przegrałeś :(", Toast.LENGTH_SHORT).show();
            przegrane += 1;
            TextView w = (TextView)findViewById(R.id.przegrane);
            w.setText("Przegrane\n" + Integer.toString(przegrane));
        }
    }
    public void Kamien(View view) {
        roll(0);
    }

    public void Papier(View view) {
        roll(1);
    }

    public void Nozyce(View view) {
        roll(2);
    }

}

File: Lista1/L1Z2kotlin/app/src/main/java/com/example/l1z2kotlin/MainActivity.kt
package com.example.l1z2kotlin

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.view.View
import android.widget.Toast
import com.example.l1z2kotlin.databinding.ActivityMainBinding
import kotlin.random.Random

class MainActivity : AppCompatActivity() {

    private lateinit var binding: ActivityMainBinding
    private var wygrane = 0;
    private var przegrane = 0;
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        binding = ActivityMainBinding.inflate(layoutInflater)
        val view = binding.root
        setContentView(view)
//        setContentView(R.layout.activity_main)
    }
    fun Roll(choice: Int) {
        var i =  Random.nextInt(3);
        if(i == 0)
        {
            binding.Odpowiedz.text= "Kamień!";
        }
        else if (i == 1)
        {
            binding.Odpowiedz.text = "Papier!";
        }
        else if(i == 2)
        {
            binding.Odpowiedz.text = "Nożyce!";
        }

        if(i == choice)
        {
            Toast.makeText(this, "Remis", Toast.LENGTH_SHORT).show();
        }
        else if(i == 0 && choice == 1)
        {
            wygrane++;
            binding.Wygrane.text = "Wygrane\n" + wygrane;
            Toast.makeText(this, "Wygrałeś!", Toast.LENGTH_SHORT).show();
        }
        else if(i == 0 && choice == 2)
        {
            przegrane++
            binding.Przegrane.text = "Przegrane\n" + przegrane;
            Toast.makeText(this, "Przegrałeś  :(", Toast.LENGTH_SHORT).show();
        }
        else if(i == 1 && choice == 0)
        {
            przegrane++
            binding.Przegrane.text = "Przegrane\n" + przegrane;
            Toast.makeText(this, "Przegrałeś  :(", Toast.LENGTH_SHORT).show();
        }
        else if(i == 1 && choice == 2)
        {
            wygrane++;
            binding.Wygrane.text = "Wygrane\n" + wygrane;
            Toast.makeText(this, "Wygrałeś!", Toast.LENGTH_SHORT).show();
        }
        else if(i== 2 && choice == 0)
        {
            wygrane++;
            binding.Wygrane.text = "Wygrane\n" + wygrane;
            Toast.makeText(this, "Wygrałeś!", Toast.LENGTH_SHORT).show();
        }
        else if( i== 2 && choice == 1)
        {
            przegrane++
            binding.Przegrane.text = "Przegrane\n" + przegrane;
            Toast.makeText(this, "Przegrałeś  :(", Toast.LENGTH_SHORT).show();
        }
   }
    fun Kamien(view: View) {
        Roll(0);
    }

    fun Papier(view: View) {
        Roll(1)
    }

    fun Nozyce(view: View) {
        Roll(2);
    }

}